File: app/src/main/java/com/cbs/sscbs/Others/Contact_us.java
package com.cbs.sscbs.Others;

import android.content.Intent;
import android.net.Uri;
import android.os.Build;
import android.os.Bundle;
import android.support.annotation.RequiresApi;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;
import android.view.View;

import com.cbs.sscbs.R;
import com.google.android.gms.maps.CameraUpdateFactory;
import com.google.android.gms.maps.GoogleMap;
import com.google.android.gms.maps.OnMapReadyCallback;
import com.google.android.gms.maps.SupportMapFragment;
import com.google.android.gms.maps.model.LatLng;
import com.google.android.gms.maps.model.Marker;
import com.google.android.gms.maps.model.MarkerOptions;

import java.util.Objects;


public class Contact_us extends AppCompatActivity implements OnMapReadyCallback{

    GoogleMap map;

    @RequiresApi(api = Build.VERSION_CODES.KITKAT)
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_contact_us);
        Toolbar toolbar = findViewById(R.id.toolbar_contact_us);
        setSupportActionBar(toolbar);
        Objects.requireNonNull(getSupportActionBar()).setDisplayHomeAsUpEnabled(true);
        getSupportActionBar().setDisplayShowHomeEnabled(true);
        findViewById(R.id.call).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(Intent.ACTION_DIAL);
                intent.setData(Uri.parse("tel:01127573447"));
                startActivity(intent);
            }
        });

        findViewById(R.id.website).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(Intent.ACTION_VIEW);
                intent.setData(Uri.parse("http://www.sscbs.du.ac.in"));
                startActivity(intent);
            }
        });

        SupportMapFragment mapFragment = (SupportMapFragment) getSupportFragmentManager()
                .findFragmentById(R.id.map);
        mapFragment.getMapAsync(this);
    }

    @Override
    public void onMapReady(GoogleMap googleMap) {
        final LatLng SSCBS = new LatLng(28.732740, 77.118748);
        googleMap.setMapType(GoogleMap.MAP_TYPE_NORMAL);
        Marker sscbs = googleMap.addMarker(new MarkerOptions()
                .position(SSCBS));
        sscbs.showInfoWindow();
        googleMap.moveCamera(CameraUpdateFactory.newLatLng(SSCBS));
        googleMap.animateCamera(CameraUpdateFactory.zoomTo(16));
    }

    @Override
    public void onPointerCaptureChanged(boolean hasCapture) {

    }
}

File: app/src/main/java/com/cbs/sscbs/Contact_us.kt
package com.cbs.sscbs

import android.content.Intent
import android.net.Uri
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.support.v7.widget.Toolbar
import android.view.View


class Contact_us : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_contact_us)
        val toolbar = findViewById<View>(R.id.toolbar_contact_us) as Toolbar
        setSupportActionBar(toolbar)
        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
        supportActionBar!!.setDisplayShowHomeEnabled(true)

        findViewById<View>(R.id.call).setOnClickListener {
            val i = Intent(Intent.ACTION_DIAL)
            val p = "tel:01122154581\n"
            i.data = Uri.parse(p)
            startActivity(i)
        }


        findViewById<View>(R.id.email).setOnClickListener {
            val sub = ""
            val body = ""
            val intent = Intent(Intent.ACTION_VIEW)
            val data = Uri.parse("mailto:sscbs@gmail.com?subject=$sub&body=$body")
            intent.data = data
            startActivity(Intent.createChooser(intent, "Send Email Using"))
        }

        findViewById<View>(R.id.website).setOnClickListener {
            val browserIntent = Intent(Intent.ACTION_VIEW)
            browserIntent.data = Uri.parse("http://www.sscbs.du.ac.in")
            startActivity(browserIntent) }

        findViewById<View>(R.id.address).setOnClickListener {
            //val uri = String.format(Locale.ENGLISH, ":%f,%f", 28.7327, 77.1188)
            val geoUri = "http://maps.google.com/maps?q=loc:$28.7327,$77.1188 ($ Dr. K.N. Katju Marg, Sector 16, Rohini, New Delhi, Delhi 110089)"
            val intent = Intent(Intent.ACTION_VIEW, Uri.parse(geoUri))
            startActivity(intent)
        }


    }
}
